### ISSUE TITLE
llm_util.schema("raw") does not work with Enums

### ISSUE THREAD

**Issue by BigJerBD** [memgraph/memgraph#2284]
**Memgraph version**
v2.19.0,

**Environment**
Memgraph Cloud

**Describe the bug**
Running:
```
CALL llm_util.schema("raw")
YIELD *
RETURN *
```
Does not work if you have an enum in your schema

**To Reproduce**

Execute those two queries:
```
CREATE ENUM TestEnum VALUES { A, B, C };
```
```
CREATE (n:TEST {v: TestEnum::A})
```
Then this command will fail:
```
CALL llm_util.schema("raw")
YIELD *
RETURN *
```

**Expected behavior**
Should return a schema with a proper enum type

```
{
   "node_props": {
      "Test": [
         {
            "property": "v",
            "type": "enum"
            // Ideally  their underlying values , such as A,B,C and the enum Id? (TestEnum)
         },
      ],
      ...
   },
}
```

**Logs**
```
[2024-08-15 16:04:07.112][Error]Logic error during mg API call: mgp_value for PropertyValue::Type::Enum doesn't exist. Contact Memgraph team under team@memgraph.com or open a new issue / comment under existing one under github.com/memgraph/memgraph.
```
**Stack Trace**
```
llm_util.schema: Traceback (most recent call last):
  File "/usr/lib/memgraph/query_modules/llm_util.py", line 168, in schema
    schema_generator = SchemaGenerator(context, output_type)
  File "/usr/lib/memgraph/query_modules/llm_util.py", line 14, in __init__
    self._generate_schema(context)
  File "/usr/lib/memgraph/query_modules/llm_util.py", line 41, in _generate_schema
    self._update_properties_dict(
  File "/usr/lib/memgraph/query_modules/llm_util.py", line 117, in _update_properties_dict
    for property_name in graph_object.properties.keys():
  File "/usr/lib/memgraph/python_support/mgp.py", line 318, in keys
    for item in self.items():
  File "/usr/lib/memgraph/python_support/mgp.py", line 295, in items
    prop = properties_it.next()
_mgp.LogicErrorError: Logic error.
```

**Comment by BigJerBD** [comment]
based on : https://github.com/memgraph/memgraph/blob/198b46434ac27080903ed33a69ac6840bcbce3c6/src/query/procedure/mg_procedure_impl.cpp#L299

It seems that it's not the only type that is not supported. Could also be relevant to eventually support Point2d , Point3d, Type::ZonedDateTime

**Comment by katarinasupe** [comment]
Hi @BigJerBD, thank you for reporting this. You are probably right because Enum, Point and ZonedDateTime were recently added data types. I'll check what we can do that. Is this a blocking issue for you now?

**Comment by katarinasupe** [comment]
Hi @BigJerBD, just wanted to update you that we released a new query - [SHOW SCHEMA INFO](https://memgraph.com/docs/querying/schema). That should be useful for your use case. Let me know if it helps!

**Comment by imilinovic** [comment]
Show schema info supports this operation on the whole graph, but we need to add new data types to mgp.

**Comment by katarinasupe** [comment]
Blocked by https://github.com/memgraph/memgraph/issues/2460

**Comment by andrejtonev** [comment]
closing since llm_utils.schema is deprecated
https://memgraph.com/docs/advanced-algorithms/available-algorithms/llm_util
